### DISCUSSION TITLE
Traefik as proxy

### DISCUSSION THREAD

**Discussion by bertmelis** [immich-app/immich#467] [Q&A] [answered]
I'm now using Immich behind  a Traefik reverse-proxy for TLS termination. I'm using Podman and Traefik doesn't automatically pick up potential services. I've configured it using files.

Now I was wondering whether it is possible to eliminate the nginx container in the Immich stack.

I'm quite a noob when it comes to Traefik. If anyone already did this, could you share your config? Or is this just not possible (I know Traefik is not the same as nginx)?

**Comment by alextran1502** [comment]
@zackpollard If you have a chance, can you share your setup here?

**Comment by NegativeFeedback** [comment, marked as answer]
working tags can be found here
https://github.com/immich-app/immich/discussions/437#discussioncomment-3429045
